src/pilot_simulation.jl

```diff
@@ -70,10 +75,12 @@ S = reverse(S, dims=2)
 ##### Set up the grid
 #####
 
-Nx = Ny = Nz = 32
-Lx = Ly = Lz = 100
-topology = (Periodic, Bounded, Bounded)
-grid = RegularCartesianGrid(topology=topology, size=(Nx, Ny, Nz) x=(0, Lx), y=(0, Ly), z=(-Lz, 0))
+Nx = Ny = 32
+Nz = 2Nx
+Lx = Ly = 1000.0
+Lz = 2Lx
```

Comment:
Should we analyze the SOSE data and set the domain depth to 2x and the domain width to 4x the maximum boundary layer depth observed in the data?